theme/site/globals/site.variables

```diff
@@ -81,8 +81,8 @@
 @simulatorBackground: @blue;
 @editorToolsBackground: @beige;
 @blocklySvgColor: @beige;
-
 @homeScreenBackground: @beige;
+@cloudCardBackground: lighten(desaturate(@teal, 56), 23);
```

Comment:
Derived the original hard-coded color of `#58AFBA` from Arcade's `@teal` using this tool: https://razorltd.github.io/sasscolourfunctioncalculator/